### PR TITLE
wxGUI/preferences: allow the user to change the font size/face of the manual page

### PR DESCRIPTION
**Pull request by tmszi**
Implementing/close feature #3268.

https://github.com/OSGeo/grass/assets/50632337/87cd9012-2d52-4704-aa72-a29f792dce96

### CODE REVIEW

File: gui/wxpython/gui_core/ghelp.py [hunk 1]
```diff
@@ -819,6 +820,32 @@ def fillContentsFromFile(self, htmlFile, skipDescription=True):
             self.loaded = False
 
 
+class ManualPageHelpWindow(HelpWindow):
```
Comment (petrasovaa) [inline gui/wxpython/gui_core/ghelp.py:823]: What is the purpose of creating a derived class, why not to implement that directly in HelpWindow?

Comment (tmszi) [reply gui/wxpython/gui_core/ghelp.py]: I wrongly thought that the base classes are used by About GRASS GIS dialog. Fixed with https://github.com/OSGeo/grass/pull/3291/commits/8184c419d915eca0f4dff3b7d8e68cc8a16cfbd3.

**Comment by cmbarton** [comment]
Just now compiled and tested. Works as intended. However, it generates a (bogus?) error in the terminal:

```
2023-12-08 15:46:28.493 python[17294:11493842] CoreText note: Client requested name ".SFNS-Regular", it will get TimesNewRomanPSMT rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
```

I picked Calibri on my Mac and it seems to show Calibri regardless of the error claiming that I'll see Times New Roman.

**Comment by cmbarton** [comment]
I just did it a second time and there is no error. Maybe once I save and it creates an entry in the GUI data file there is no longer an error.

**Pull request merged**